lib/index.js

```diff
@@ -103,6 +103,14 @@ module.exports = (bookshelf) => {
                         previousAttrs = model.saveAttributes();
                         // modify columns from the request payload
                         const toSave = _.extend({}, existing, data);
+                        // set updated_at column automatically
+                        if (model.hasTimestamps) {
+                            let updatedAtColumn = 'updated_at';
+                            if (Array.isArray(model.hasTimestamps) && model.hasTimestamps.length > 1) {
+                                updatedAtColumn = model.hasTimestamps[1];
+                            }
+                            model.set(updatedAtColumn, undefined);
```

Comment:
does setting the value to 'undefined' update the timestamp to the current time?

Reply:
I believe it should since Bookshelf defaults to current time if not set, I'll merge this